prow/cmd/deck/runlocal

```diff
@@ -24,4 +24,4 @@ curl "$HOST/data.js?var=allBuilds" > data.js
 curl "$HOST/tide.js?var=tideData" > tide.js
 curl "$HOST/plugin-help.js?var=allHelp" > plugin-help.js
 curl "$HOST/pr-data.js" > pr-data.js
-bazel run --run_under="cd $DIR && " //prow/cmd/deck:deck -- --run-local --static-files-location=./static --template-files-location=./template --config-path ../../config.yaml
+bazel run //prow/cmd/deck:deck -- --run-local=${DIR}/localdata --static-files-location=./prow/cmd/deck/static --template-files-location=${DIR}/template --config-path ${DIR}/../../config.yaml
```

Comment:
in recent versions of Bazel, `bazel run` sets `BUILD_WORKSPACE_DIRECTORY` and `BUILD_WORKING_DIRECTORY`, though I'm not sure that's relevant here.

I guess I'm a little confused why everything except `--static-files-location` is using `${DIR}` though.

Reply:
In the case of `--template-files-location`, there is no good reason; I'll change it.

For `--run-local` (which I agree I should rename), the pseudo-static content isn't pulled in by bazel and I see no particular reason to do so. It references $DIR because that's where we happen to put the files (it could just as well be /tmp, and perhaps it should be now that we have dropped the requirement that they live among the static files).

Similarly, for `--config-path`, the config file isn't part of the build, and I don't think it should be.

(`--static-files-location` _must_ refer to the bazel output, which is why it is the only one to do so.)